projects/packages/paypal-payments/src/block/simple-payments.php

```diff
@@ -0,0 +1,102 @@
+<?php
+/**
+ * Pay with PayPal block (aka Simple Payments).
+ *
+ * @since 9.0.0
+ *
+ * @package automattic/jetpack
+ */
+
+namespace Automattic\Jetpack\PaypalPayments\SimplePayments;
+
+use Automattic\Jetpack\Blocks;
+use Jetpack_Simple_Payments;
+use WP_Post;
+
+const FEATURE_NAME = 'simple-payments';
+const BLOCK_NAME   = 'jetpack/' . FEATURE_NAME;
+
+/**
+ * Registers the block for use in Gutenberg
+ * This is done via an action so that we can disable
+ * registration if we need to.
+ */
+function register_block() {
+	Blocks::jetpack_register_block(
+		__DIR__,
+		array(
+			'render_callback' => __NAMESPACE__ . '\render_block',
+			'plan_check'      => true,
+		)
+	);
+}
+add_action( 'init', __NAMESPACE__ . '\register_block' );
+
+/**
+ * Pay with PayPal block dynamic rendering.
+ *
+ * @param array  $attr    Array containing the block attributes.
+ * @param string $content String containing the block content.
+ *
+ * @return string
+ */
+function render_block( $attr, $content ) {
+	// Do nothing if block content is a `simple-payment` shortcode.
+	if ( preg_match( '/\[simple-payment(.*)]/', $content ) ) {
+		return $content;
+	}
+
+	// Keep content as-is if rendered in other contexts than frontend (i.e. feed, emails, API, etc.).
+	if ( ! jetpack_is_frontend() ) {
+		return $content;
+	}
+
+	$simple_payments = Jetpack_Simple_Payments::get_instance();
```

Comment:
You'll need to use `Automattic\Jetpack\Paypal_Payments\Simple_Payments` here instead of the class from the Jetpack plugin.

Reply:
Good catch here as well. Fixed in [91f2474](https://github.com/Automattic/jetpack/pull/43413/commits/91f2474b92c094ffca5420d1be9dc27a88678398).